django_slack/backends.py

```diff
@@ -22,11 +24,13 @@ def send(self, url, data):
 
         self.validate(r.headers['Content-Type'], r.text)
 
+
```

Comment:
sorry ... looks like those were added by my editor because of pep8 ( 2 blank lines between classes)